BlueprintUI/Sources/Layout/Layout.swift

```diff
@@ -1,29 +1,125 @@
-import UIKit
+import CoreGraphics
 
-/// Conforming types can calculate layout attributes for an array of children.
-public protocol Layout {
+/// A type that defines the geometry of a collection of elements.
```

Comment:
Love these docs; nice work!

Reply:
I should mention that most of these were cribbed from SwiftUI's docs, and then adapted to Blueprint.

Reply:
You may want to see for yourself: [Layout](https://developer.apple.com/documentation/swiftui/layout)

I hadn't really considered that borrowing too much documentation might be a problem (since we are intentionally trying to mimic the API). A citation is easy enough.

Reply:
The first few paragraphs look like a pretty direct copy. We should cite the source in a line like "Borrowed with little modification from [X](link)."

Reply:
How's [this](https://github.com/square/Blueprint/pull/446/commits/1d3548120da3fd07b63d39871c136244a88c757c#diff-28d36c422d80ceb0a832b0a43375d60f4602ad7f37d4d4199a02bc99dfa4ded9)?